### DISCUSSION TITLE
Cache is always missed in non-monorepo

### DISCUSSION THREAD

**Discussion by thisiszinger** [vercel/turborepo#5109] [Help]
Hi! I'm trying to use Turborepo to speed up build (and other task) times in my non-monorepo Next.js app setup. I have `turbo` (1.9.9) installed as a dev dependency and when I run it locally with `pnpm turbo build` I am getting cache hits (after the first run). However, when running on Vercel I am consistently getting cache misses. I have listed all environment variables under `globalEnv` in `turbo.json` (also using `eslint-config-turbo` to catch anything I might've missed). I have my Vercel build command overridden and set to `turbo build`. I also have remote caching enabled. All other settings are default. Would appreciate any help, thanks!

**Comment by mehulkar** [comment]
Could you try using Run Summaries to compare two deployments and see if any of the inputs have changed? https://vercel.com/docs/monorepos/turborepo#debug-turborepo-cache-misses

**Comment by ajayvignesh01** [comment]
Hey, were you able to figure it out? Running into the same issue in our single package workspace
